code/game/machinery/iv_drip.dm

```diff
@@ -111,12 +112,19 @@
 			amount = min(amount, 4)
 			// If the beaker is full, ping
 			if(amount == 0)
-				if(prob(5)) visible_message("\The [src] pings.")
+				if(prob(5)) 
+					visible_message("\The [src] pings.")
 				return
 
+			if(T.get_blood_volume() < 90 && !blood_message_sent)
+				visible_message("\icon[src] \The <b>[src]</b> flashes a warning light!")
+				playsound(src, 'sound/machines/buzz-two.ogg', 50)
+				blood_message_sent = TRUE
+
 			var/datum/reagent/B = T.take_blood(beaker,amount)
+			B.volume += amount * 0.8
```

Comment:
I'm not so sure about this. Just... make blood bags smaller. We already have enough infinite chem duplication glitches as it is, we don't need to intentionally add one.